guides/_layouts/default.html

```diff
@@ -3,45 +3,55 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    {% if page.section contains "GraphQL" %}
-      <title>{{ page.section }} - {{ page.title }}</title>
+    {% if page.target == "dash" %}
+      <title>{{ page.title }}</title>
+      <link rel="stylesheet" href="{{ "/css/main.css" | relativize_url }}">
```

Comment:
If I understood right, this outputs `css/main.css` on the homepage, and `../css/main.css` on child pages, is that right? And I guess it's needed because we don't control the path where Dash renders the HTML?